src/sets.jl

```diff
@@ -184,6 +184,28 @@ constant(s::EqualTo) = s.value
 constant(s::GreaterThan) = s.lower
 constant(s::LessThan) = s.upper
 
+"""
+    NormInfinityCone(dimension)
+
+The epigraph of the L∞ norm function ``\\{ (t,x) \\in \\mathbb{R}^{dimension} : t \\ge || x ||_∞ = \\max_i |x_i| \\}`` of dimension `dimension`.
```

Comment:
Shouldn't it be \infty in LaTeX ?

Reply:
sure. is there a way to preview the latex output?